bindings-go/apis/v2/componentdescriptor.go

```diff
@@ -427,3 +437,35 @@ func (o *ComponentReference) GetIdentity() Identity {
 func (o *ComponentReference) GetIdentityDigest() []byte {
 	return o.GetIdentity().Digest()
 }
+
+// DigestSpec defines the digest and algorithm.
+// +k8s:deepcopy-gen=true
+// +k8s:openapi-gen=true
+type DigestSpec struct {
+	Algorithm string `json:"algorithm"`
+	Value     string `json:"value"`
+}
+
+// SignatureSpec defines the signature and algorithm.
+// +k8s:deepcopy-gen=true
+// +k8s:openapi-gen=true
+type SignatureSpec struct {
+	Algorithm string `json:"algorithm"`
+	Data      string `json:"data"`
+}
```

Comment:
why is it called `value` in the digest and in the signature the data field is called `data`.
I think we should use one naming